etna/images/models.py

```diff
@@ -5,108 +5,135 @@
 from modelcluster.models import ClusterableModel
 from wagtail.fields import RichTextField
 from wagtail.images.models import AbstractImage, AbstractRendition
+from wagtail.search import index
 
 from etna.records.fields import RecordField
 
 
-class TranscriptionLanguageChoices(models.TextChoices):
-    OLD_ENGLISH = "Old English", _("Old English")
-    MIDDLE_ENGLISH = "Middle English", _("Middle English")
-    EARLY_MODERN_ENGLISH = "Early Modern English", _("Early Modern English")
-    MODERN_ENGLISH = "Modern English", _("Modern English")
-    LATIN = "Latin", _("Latin")
+class TranscriptionHeadingChoices(models.TextChoices):
+    TRANSCRIPT = "transcript", _("Transcript")
+    PARTIAL_TRANSCRIPTION = "partial-transcript", _("Partial transcript")
+
+
+class TranslationHeadingChoices(models.TextChoices):
+    TRANSLATION = "translation", _("Translation")
+    MODERN_ENGLISH = "modern-english", _("Modern English")
```

Comment:
Do these make something similar to Enums? I saw you use it as `default=TranscriptionHeadingChoices.TRANSCRIPT`, which I understand is pointing to TRANSCRIPT here - but what does that do to the field where it's referenced?

Reply:
Yep, these are Enum subclasses provided by Django! When you do `TranscriptionHeadingChoices.TRANSCRIPT`, the interface is such that it will always return the value ("transcript") and not the (value, label) tuple you use in the definition.